lib/acts_as_taggable_on_mongoid/taggable/tag_type_definition/attributes.rb

```diff
@@ -62,14 +58,67 @@ def remove_unused_tags
         alias force_parameterize? force_parameterize
         alias remove_unused_tags? remove_unused_tags
 
+        def tagger?
+          instance_variable_defined?(:@tagger)
+        end
+
+        def tag_list_uses_default_tagger?
+          return false if !tagger? || default_tagger_method.blank?
+
+          tagger_params.fetch(:tag_list_uses_default_tagger) { false }
+        end
+
+        def default_tagger_method
+          return nil unless tagger?
+
+          tagger_params[:default_tagger]
+        end
+
+        def taggable_default(taggable)
+          taggable_default = default
+
+          return unless taggable_default.present?
```

Comment:
Refers to 'taggable_default' more than self (maybe move it to another class?) (FeatureEnvy)